safe-global/web-core
# fix: reset error boundary after navigation

## What it solves

Resolves #1539

## How this PR fixes it

A standard link is used instead of an internally routed one.

## How to test it

Navigate to `/apps?safe=gor:0x4127839cdf4F73d9fC9a2C2861d8d1799e9DF40C&appUrl=appUrl%3Djavascript%3AstartHack%28%29)` and click "Go Home". Observe that the `/welcome` page is opened and error stack is reset.
